File: 백준/Bronze/2480. 주사위 세개/주사위 세개.js
const fs = require('fs');

const readLine = fs.readFileSync('/dev/stdin').toString().trim();
const oppertunity = readLine.split(' ').map(Number);
const resultCount = Array(7).fill(0);
let maxCount = 0;
let maxNum = 0;
let manyNum = 0;

for (var i = 0; i < oppertunity.length; i++) {
    let num = oppertunity[i];
    resultCount[num]++;

    if (maxCount < resultCount[num]) {
        maxCount = resultCount[num];
    }

    if (maxNum < num) {
        maxNum = num;
    }

    if (maxCount === resultCount[num]) {
        manyNum = num;
    }
}

let awards = 0;
if (maxCount === 1) {
    awards = maxNum * 100;
} else if (maxCount === 2) {
    awards = 1000 + manyNum * 100;
} else {
    awards = 10000 + manyNum * 1000;
}

console.log(awards);

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
import sys

readline = sys.stdin.readline
resultCount = [0 for _ in range(7)]
awards = 0

opportunity = list(map(int, readline().split()))
for num in opportunity :
    resultCount[num] += 1

maxCount = max(resultCount)
manyNum = resultCount.index(maxCount)
maxNum = [i for i, count in enumerate(resultCount) if count != 0][-1]

if maxCount == 1 :
    awards = maxNum * 100
elif maxCount == 2 :
    awards = 1000 + manyNum * 100
else :
    awards = 10000 + manyNum * 1000

print(awards)